### PR TITLE
Various bug fix and update to Readme.MD

### PR DESCRIPTION
**Pull request by sim0mo**
This PR fixes some minor bugs :
- In noSing mode, every player now sees the text saying "Listen !"
- If a user modifies a playlist, the button in AddPlaylistActivity.kt says "update playlist" instead of "Add playlist"
- Fix the color of a text view in LobbyActivity
- Speed up a bit TextToSpeech voice

In addition to this, this PR finally give a first improvement to Readme.md . Of course it can be improved later, but it will temporarily look prettier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,47 @@
...
+- Players can put their playlist on the cloud in order to use them in a game with their friends. 
+- A voice channel is setup to be able to sing and be heared, and moreover to hear the reactions of your friends ! 
+- The app is available in english and french.
+  
+
+
+Voice channel : https://www.agora.io/
 
-&copy; Simon Armand Marti
+Lyrics provider : https://www.musixmatch.com/
```
Comment (BartLucien) [inline README.md:47]: You may want to add that spotify is used to import playlist  and FirestoreDatabase for the database.

File: app/src/main/java/com/github/fribourgsdp/radio/game/handler/PlayerGameHandler.kt [hunk 2]
```diff
@@ -58,6 +58,7 @@ class PlayerGameHandler(
             // It's not null when there is one.
             songToGuess = snapshot.getString("current_song")
 
+            Log.println(Log.ASSERT, "UP", "handle snapshot : noSing=${noSing}")
```
Comment (BartLucien) [inline app/src/main/java/com/github/fribourgsdp/radio/game/handler/PlayerGameHandler.kt:61]: Is this still needed?

**Review by VictorSchneuwly — APPROVED** [review]
Same remarks as @BartLucien but outside of that, it's all good to me. 🦦

**Review by BartLucien — APPROVED** [review]
Seems good to me.

**Pull request merged**